pkgs/applications/office/ledger-autosync/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, python3Packages, fetchurl }:
```

Comment:
Drop `lib` here as it will be unused after the changes done in `meta` are applied.